editions/tw5.com/tiddlers/demonstrations/NestedLists.tid

```diff
@@ -0,0 +1,44 @@
+title: NestedLists
+tags: ListWidget
+created: 20150106180000000
+modified: 20150106180000000
+
+The following sidebar tabs give examples of grouped lists created by nesting.
+
+!! Types Tab
+
+The list in the [[Types|$:/core/ui/MoreSideBar/Types]] tab is generated with this markup:
+
+```
+<$list filter="[!is[system]has[type]each[type]sort[type]]">
```

Comment:
I think that the macro name `<<example>>` is too generic, better leave that for users own macros. I'd call this macro `<<plain-text tiddler>>`. Also I just realised that we should use the codeblock widget. That way the text will be syntax highlighted if the plugin is installed:

```
\define plain-text(tiddler)
<$codeblock language={{$tiddler$!!type}}>
<$text text={{$tiddler$}}/>
</$codeblock>
```